.config/mise.toml

```diff
@@ -0,0 +1,95 @@
+#:schema https://mise.jdx.dev/schema/mise.json
+
+[vars]
+bacon_version = "3.16.0"
+cargo_insta_version = "1.43.1"
+cargo_nextest_version = "0.9.103"
+rust_version = "1.85"
+uv_version = "0.8.13"
+
+[tools]
+# We need cargo-binstall so that Mise would download "cargo:" tools instead of building them.
+"cargo-binstall" = "latest"
+
+[tasks."build:cli-reference"]
+description = "Build the command line reference (cli/tests/cli-reference@.md.snap)"
+tools.rust = "{{vars.rust_version}}"
+tools."cargo:cargo-insta" = "{{vars.cargo_insta_version}}"
+run = "cargo insta test --accept --workspace -- test_generate_md_cli_help"
+
+[tasks."build:debug"]
+alias = "Build jj in debug mode"
+description = "Build jj"
+tools.rust = "{{vars.rust_version}}"
+run = "cargo build"
+
+[tasks."build:docs"]
+description = "Build documentation into rendered-docs/ for offline use"
+tools.uv = "{{vars.uv_version}}"
+run = "uv run mkdocs build"
+
+[tasks."build:release"]
+description = "Build jj in release mode"
+tools.rust = "{{vars.rust_version}}"
+run = "cargo build --release"
+
+
+[tasks."check:clippy"]
+description = "Lint the code with Clippy"
+tools.rust = "{{vars.rust_version}}"
+run = "cargo clippy --workspace --all-targets"
+
+[tasks."check:format"]
+description = "Check the code format"
+tools.rust = "nightly"
+run = "cargo fmt --check"
+
+[tasks."check:test"]
+alias = "test"
+description = "Run the tests"
+tools.rust = "{{vars.rust_version}}"
+tools."cargo:cargo-insta" = "{{vars.cargo_insta_version}}"
+tools."cargo:cargo-nextest" = "{{vars.cargo_nextest_version}}"
+env.NEXTEST_STATUS_LEVEL="fail"
```

Comment:
Looking at [the docs](https://nexte.st/docs/reporting/#status-levels), I think it might be better to use "slow" here. I think it is better to show slow and retried tests by default (though both are rare in the `jj` repo).

>  It might not be to everyone's taste, though. Any opinion?

I haven't tested it myself or done my own research, but trying this out sounds very reasonable to me. I wasn't aware of this option myself, but I probably want it. I'm curious whether we can find an option people will be happy with and, if not, whether people would find it simple enough to override it via `mise.local.toml`.

Reply:
You could file an FR for nextest! I meant to, but I couldn't immediately decide what UI would be better exactly.